sagemaker-training-compiler/tensorflow/single_gpu_single_node/vision-transformer.ipynb

```diff
@@ -0,0 +1,642 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # TODO: Change how TrainingCompilerConfig is used after SDK release
</pre><p>Can this TODO item be addressed now?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3458/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>+1</p>

Reply:
Not yet: Tracking https://github.com/aws/sagemaker-python-sdk/pull/3156